coalib/results/Result.py

```diff
@@ -36,7 +36,7 @@ def __init__(self,
                  severity: int=RESULT_SEVERITY.NORMAL,
                  additional_info: str="",
                  debug_msg="",
-                 diffs: (dict, None)=None,
+                 diffs: (Iterable, None)=None,
```

Comment:
Don't you need to import Iterable anywhere ? (Same for the below use of Iterable)